src/test/unit_tests/jest_tests/test_nc_account_cli.test.js

```diff
@@ -103,6 +103,33 @@ describe('manage nsfs cli account flow', () => {
             assert_account(account_symlink, account_options);
         });
 
+        it('should fail - cli create account invalid new_buckets_path (leading space in new_buckets_path)', async () => {
+            const { type, name, uid, gid, new_buckets_path } = defaults;
+            const account_options = { config_root, name, uid, gid}; // will add new_buckets_path with leading space later
+            const action = ACTIONS.ADD;
+            const command = create_command(type, action, account_options);
+            const res = await exec_manage_cli_add_leading_space_option(command, 'new_buckets_path', new_buckets_path, 's'); // only space
+            expect(JSON.parse(res.stdout).error.code).toBe(ManageCLIError.InvalidArgument.code);
+        });
+
+        it('should fail - cli create account invalid new_buckets_path (leading space in quoted new_buckets_path)', async () => {
+            const { type, name, uid, gid, new_buckets_path } = defaults;
+            const account_options = { config_root, name, uid, gid}; // will add quoted new_buckets_path with leading space later
+            const action = ACTIONS.ADD;
+            const command = create_command(type, action, account_options);
+            const res = await exec_manage_cli_add_leading_space_option(command, 'new_buckets_path', new_buckets_path, 'sq'); // space then quote
+            expect(JSON.parse(res.stdout).error.code).toBe(ManageCLIError.InvalidArgument.code);
+        });
+
+        it('should fail - cli create account invalid new_buckets_path (quoted leading space in new_buckets_path)', async () => {
+            const { type, name, uid, gid, new_buckets_path } = defaults;
+            const account_options = { config_root, name, uid, gid}; // will add new_buckets_path with quoted leading space later
+            const action = ACTIONS.ADD;
+            const command = create_command(type, action, account_options);
+            const res = await exec_manage_cli_add_leading_space_option(command, 'new_buckets_path', new_buckets_path, 'qs'); // quote then space
+            expect(JSON.parse(res.stdout).error.code).toBe(ManageCLIError.InvalidAccountNewBucketsPath.code);
```

Comment:
The error here is about "InvalidAccountNewBucketsPath", but the added code is about the new error "InvalidArgument", so I find this case confusing.

Reply:
Here, I have added a test just to verify if the space is also quoted with the value then we should get different error.

Reply:
I would add a comment as it is a bit confusing.